crates/nu-cli/src/commands/last.rs

```diff
@@ -70,20 +70,17 @@ async fn last(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStr
     };
 
     let count = rows_desired as usize;
+    let rows_to_show = v.len() - count;
 
-    let k = if count < v.len() {
-        v.len() - count
-    } else {
-        return Ok(futures::stream::iter(v).to_output_stream());
-    };
+    let iter = v.into_iter().enumerate().filter_map(move |(i, value)| {
+        if rows_to_show <= i {
```

Comment:
I think you want this to be `>`, not `<=`, but even better: https://docs.rs/futures/0.3.5/futures/stream/trait.StreamExt.html#method.skip

Reply:
Good catch!  A nice, definitely am using `skip()` now